Options.py

```diff
@@ -973,7 +973,13 @@ def from_any(cls, data: PlandoTextsFromAnyType) -> Self:
                     if random.random() < float(text.get("percentage", 100)/100):
                         at = text.get("at", None)
                         if at is not None:
+                            if isinstance(at, dict):
+                                at = random.choices(list(at.keys()),
+                                                    weights=list(at.values()), k=1)[0]
                             given_text = text.get("text", [])
+                            if isinstance(given_text, dict):
+                                given_text = random.choices(list(given_text.keys()),
+                                                            weights=list(given_text.values()), k=1)
```

Comment:
if `text["at"]` is an empty dict, or `text["text"]` is an empty dict, this will crash with an error message that is not clear to users.